src/main/scala/edu/knowitall/srlie/SrlExtraction.scala

```diff
@@ -196,6 +216,16 @@ object SrlExtraction {
     def tokenSpan = span
 
     override def hashCode = intervals.hashCode * 39 + tokens.hashCode
+    def canEqual(that: Any): Boolean = that.isInstanceOf[Context]
```

Comment:
Copied it from Context, and updated the equals implementation but not canEqual. I'll fix.